test/mailers/user_mailer_test.rb

```diff
@@ -18,7 +18,7 @@ class UserMailerTest < ActiveSupport::TestCase
     mail = UserMailer.contact(sender: sender, resource_id: receiver.id, text: text)
 
     mail.must deliver_to receiver.email
-    mail.subject.must_equal('[Fairmondo] ein/e Nutzer*in hat eine Frage an Dich')
+    mail.subject.must_equal('[Fairmondo] #{sender.nickname} hat eine Frage an Dich')
```

Comment:
For interpolation in ruby you need double quotes `""` instead of single quotes `''` that is the reason for the failed CI as well.